cypress/e2e/RoundTrips/SimpleProject.feature

```diff
@@ -0,0 +1,473 @@
+    When I set on '[data-cy="search-diagram-input"]' text '1'
+    Then I expect '[data-cy="diagram-path_{{secondModelFolder}}"]' not exists
+    And  I expect '[data-cy="diagram-path_\.github/workflows/{{thirdModelName}}"]' not exists
+
+    # 801 Clear filter text and verify all diagrams are present
+    When I set on '[data-cy="search-diagram-input"]' text ' '
+    Then I expect '[data-cy="diagram-path_{{secondModelFolder}}"]' exists
+    And  I expect '[data-cy="diagram-path_\.github/workflows/{{thirdModelName}}"]' exists
+
+    # 802 # Select Infrastructure tag and verify only all Infrastructure diagrams are present
+    When I select '[data-cy="select-checkbox_Infrastructure"]' in '[data-cy="diagram-tag-select"]'
+    And  I click on '[data-cy="diagram-tag-select"]'
+    Then I expect '[data-cy="diagram-path_{{secondModelFolder}}"]' exists
+    And  I expect '[data-cy="diagram-path_\.github/workflows/{{thirdModelName}}"]' not exists
+
+    # 803 Select CI tag and verify all diagrams are present
+    When I select '[data-cy="select-checkbox_CI"]' in '[data-cy="diagram-tag-select"]'
+    And  I click on '[data-cy="diagram-tag-select"]'
+    Then I expect '[data-cy="diagram-path_{{secondModelFolder}}"]' exists
+    And  I expect '[data-cy="diagram-path_\.github/workflows/{{thirdModelName}}"]' exists
+
+    # 804 Unselect Infrastructure tag and verify only all CI diagrams are present
+    When I click on '[data-cy="chip_Infrastructure"] i[aria-label="Remove"]'
+    Then I expect '[data-cy="diagram-path_{{secondModelFolder}}"]' not exists
+    And  I expect '[data-cy="diagram-path_\.github/workflows/{{thirdModelName}}"]' exists
+
+    # 805 Unselect CI tag and verify all diagrams are present
+    When I click on '[data-cy="chip_CI"] i[aria-label="Remove"]'
+    Then I expect '[data-cy="diagram-path_{{secondModelFolder}}"]' exists
+    And  I expect '[data-cy="diagram-path_\.github/workflows/{{thirdModelName}}"]' exists
+
+    When  I click on '[data-cy="diagram-path_{{secondModelFolder}}"]'
+
+    ## Select 'terrator-plugin' library
+    And  I click on '[data-cy="component-definitions-item_terrator-plugin"]'
+    Then I expect '[data-cy="component-definitions-item_terrator-plugin"].selected' exists
+    And  I expect '[data-cy="component-definition-grid"]' exists
+    And  I expect '[data-cy="component-definition-grid"] [class*="component-definition-card"]' appear 18 times on screen
+
+    ## 900 Plugin test installed in component definitions list (Draw view) should not create project configuration file (Text view)
+    When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
+    And  I wait 1 second
+    Then I expect '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="true"] [class="block"]' is 'Text'
+
+    When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
+    And  I wait 1 second
```

Comment:
Maybe add a line to check that the project configuration file is not created (to match the title of the scenario)
ex `I expect '[data-cy="file_leto-modelizer.config.json"]' not exists`